azurelinuxagent/common/protocol/extensions_goal_state_from_extensions_config.py

```diff
@@ -575,9 +576,13 @@ def __parse_and_add_extension_settings(settings_node, name, extension, depends_o
 
     def supports_encoded_signature(self):
         """
-        Returns bool indicating if the ExtensionsConfig API supports the 'encoded_signature' extension property. Should always return True.
+        Return True if the ExtensionsConfig API supports the 'encoded_signature' extension property and agent is
```

Comment:
supports_encoded_signature can trigger a network request. it should be reflected in its name